CHANGELOG.md

```diff
@@ -2,9 +2,13 @@
 
 * Allow specifying check-interval higher than 0 together with no limit configuration
 * Set default check-interval to 60s
+
+## 0.3.1
```

Comment:
I think both changes should go into 0.3.1 and you need anyway both for the operator PR.